pallets/parachain-staking/src/lib.rs

```diff
@@ -1871,6 +1907,10 @@ pub mod pallet {
 			let mut round = <Round<T>>::get();
 			let (now, first, old) = (round.current, round.first, round.length);
 			ensure!(old != new, Error::<T>::NoWritingSameValue);
```

Comment:
This should be `new >= old` and the error should change.

Reply:
Why would we want to reject anything `>= old`? FYI, we now have in addition to the `!= new` check:
```rs
ensure!(new >= <TotalSelected<T>>::get(), ... );
```

Reply:
We want to reject any new < old because round length can only be increased after this PR (according to a discussion between Alan and you that Alan referenced during the code review on Friday).

Reply:
I think the only constraint is that it must be more than the number of collators for the round being paid out (e.g. `round.current - RewardPaymentDelay`), which is a difficult constraint to ensure. We're not enforcing this constraint, in part because we have control over it (e.g. governance), and it goes away if we move to manual payouts.

So a safe rule of thumb would be that new always > old, but that doesn't have to be the case. I suppose we could artificially enforce it in code, though, especially if we intend to abide by that rule anyway... :thinking: